achievements/api-platform.md

```diff
@@ -1 +1,49 @@
+# API Platform Project achievements
 
+This weekend we were multiple contributors to API Platform, some of the core team were here too to help in the hackathon.
```

Comment:
```suggestion
This weekend we were multiple API Platform contributors, some of the core team were here too to help in the hackathon.
```

Reply:
I'd suggest:
During the hackathon, multiple API Platform contributors were present, some of theme being core team members and available to help others.